qemu/tests/qemu_guest_agent.py

```diff
@@ -722,6 +722,40 @@ def _action_after_fsfreeze(self, *args):
                                  " write file")
 
     @error.context_aware
```

Comment:
Just confirm, Do I need remove all @error.context_aware and use test.error instead of error.TestError in this qemu_guest_agent.py? 

I always @error.context_aware works.